pandas/tests/indexes/multi/test_base.py

```diff
@@ -0,0 +1,938 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import pandas as pd
+import pandas.util.testing as tm
+import pytest
+from pandas import (CategoricalIndex, DatetimeIndex, Float64Index, Index,
+                    Int64Index, IntervalIndex, MultiIndex, PeriodIndex,
+                    RangeIndex, Series, TimedeltaIndex, UInt64Index, compat,
+                    isna)
+from pandas._libs.tslib import iNaT
+from pandas.compat import PY3
+from pandas.core.indexes.base import InvalidIndexError
+from pandas.core.indexes.datetimelike import DatetimeIndexOpsMixin
+
+
+def verify_pickle(indices):
```

Comment:
don't create functions like this, inline it instead (unless there is a lot of usage about this, but indicate that)